src/main/resources/templates/c_plus_plus/test/tests/TestCatch2.py

```
@@ -0,0 +1,36 @@
+        self.pWrap = self._createPWrap([join(".", self.executable), "--success", "--reporter", f"JUnit::out={outputFilename}", "--reporter", "console::out=-::colour-mode=none"], self.executionDirectory)
+        self._startPWrap(self.pWrap)
+        self.pWrap.waitUntilTerminationReading()
+
+        retCode: int = self.pWrap.getReturnCode()
+        # parse XML output and append it to the results
+        try:
+            catchXmlRoot: Et = Et.parse(join(self.executionDirectory, outputFilename))
+            catchXmlSuite: Et.Element = catchXmlRoot.find("testsuite")
+            self.additionalSuites.append(catchXmlSuite)
+            printTester(f"Appended {catchXmlSuite}")
+        except Exception as e:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)